### PR TITLE
Upgrade go and its dependencies

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>

### CODE REVIEW

File: Dockerfile.prow [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/stolostron/builder:go1.18-linux as builder
+FROM registry.ci.openshift.org/stolostron/builder:go1.20-linux as builder
```
Comment (lennysgarage) [inline Dockerfile.prow:1]: I have a feeling the same issue with the other two CLC repos will happen here. I recommend just upgrading to 1.19.

Comment (chenz4027) [reply Dockerfile.prow:1]: Yep @lennysgarage you also ran into the same issue?

Comment (lennysgarage) [reply Dockerfile.prow]: Yeah same issue here, https://github.com/stolostron/provider-credential-controller/pull/64

**Pull request merged**